pom.xml

```diff
@@ -143,7 +143,7 @@
     <hadoop.version>3.3.1</hadoop.version>
     <hadoop.bin.path>${basedir}/${hive.path.to.root}/testutils/hadoop</hadoop.bin.path>
     <hamcrest.version>1.3</hamcrest.version>
-    <hbase.version>2.0.0-alpha4</hbase.version>
+    <hbase.version>2.5.5</hbase.version>
```

Comment:
may be you need 2.5.5-hadoop3 here 
https://mvnrepository.com/artifact/org.apache.hbase/hbase-client/2.5.5-hadoop3

can try that, if the test passes post that

Reply:
Thanks for suggestion. I have used the 2.5.5 and 2.5.5-hadoop3 version, but both of the two failed.
I think Hbase2.5.5 has some compatibility issues with Hadoop3.3.1, and i have filed a Hbase ticket https://issues.apache.org/jira/browse/HBASE-28061. And i also find another ticket about Hbase2.5.5&Hadoop3.3.1 https://issues.apache.org/jira/browse/HBASE-28053.  I hope that can get some valuable info from Hbase community.

Reply:
Any pointers, here I see you said on the ticket hbase built with 3.3.1 works fine

Reply:
I think 2.5.6 is released, we can give that a try? HBASE-28061 was fixed in 2.5.6

Reply:
Sure, Changed hbase version to 2.5.6. Thanks for reminding!